benchmark/src/main/scala/PaigesBenchmark.scala

```diff
@@ -41,10 +42,36 @@ class PaigesBenchmark {
 
   @Benchmark
   def fill0(): String =
-    Doc.fill(Doc.text(","), strs.map(Doc.text)).render(0)
+    Doc.fill(Doc.line, strs.map(Doc.text)).render(0)
 
   @Benchmark
   def fill100(): String =
-    Doc.fill(Doc.text(","), strs.map(Doc.text)).render(100)
+    Doc.fill(Doc.line, strs.map(Doc.text)).render(100)
```

Comment:
not using line here is kind of a worst case. The original fill didn't even have this arg, it always used line.